### ISSUE TITLE
Capitalization: "User Agent" or "user agent"

### ISSUE THREAD

**Issue by frivoal** [w3c/csswg-drafts#5200]
CSS specs are inconsistent as to whether user agent should be capitalized "user agent" or "User Agent". While this is clearly not a major issue, it would be nice to be consistent. A naive bit of grepping suggests that the all lower-case variant is more common, so I'd suggest we align the rest to that.

```
csswg-drafts$ grep "User Agent" */Overview.bs |wc -l
    155
csswg-drafts$ grep "user agent" */Overview.bs  | wc -l
    588
csswg-drafts$ grep "[U|u]ser agent" */Overview.bs  | wc -l
    686
```
(The last one is a crude attempt to capture the all lowercase variant including when it's at the start of a sentence, and capitalized for that reason.)

**Comment by frivoal** [comment]
s/The first one/The last one/

**Comment by frivoal** [comment]
A search an replace to capitalized is easier to perform though. And it is a special term, so having it capitalized does seem more appropriate, even if it is currently more rare. Anyway, Agenda+ to see if there's any appetite on unifying to one or the other.

**Comment by tabatkins** [comment]
Seems fine to unify. The W3C style guide mandantes lowercase, tho <https://w3c.github.io/manual-of-style/#Terms>

**Comment by frivoal** [comment]
Fixed by 32d265162ed6bf13e019888b8eca8e8c828b6afa
Run this command once in a while to fix automatically fixable instances and to be informed of those that need manual handling:
`bin/ua-capitalization.sh */Overview.bs`
